client/packages/invoices/src/StockOut/Components/StockOutAlerts.tsx

```diff
@@ -24,7 +24,15 @@ export const StockOutAlerts = ({
       ? allocationAlerts
       : [];
 
-  if (alerts.length === 0) return null;
+  return <AllocationAlerts allocationAlerts={alerts} />;
```

Comment:
leaving existing component for now, for prescriptions. using the display only one below for outbound